pages/implementation/reference/javascript.mdx

```diff
@@ -0,0 +1,201 @@
+
+### Handling tracking plan changes
+
+When tracking plan changes, you generate the file again and swap the old file with the new one. Avo manages the merge conflicts inside the file.
+Then, if the existing events changed, you'll have to update your Avo functions calls according to the new tracking plan to pass the runtime checks in the Avo file.
+You'll also get access to the new events defined in your tracking plan after the previous generated code update.
+
+### Platforms
+
+Avo can generate JavaScript code targeted at the following platforms
+
+- Web
+- React Native 
+- Node.js
+
+### Destinations
+
+You can send your data from Javascript to any event based analytics destionation, including
+
+- Amplitude
+- FacebookAnalytics
+- FullStory
+- Mixpanel
+- Mixpanel
+- Permutive
+- Segment
+- Snowplow
+- ZendeskConnect
+- Adobe Analytics
+- Apptelemetry
+- Rudderstack
+- Freshpaint
+- Posthog
+- Google Analytics 4 / Firebase Analytics
+- Heap
+- Keen
+- Kissmetrics
+- LaunchDarkly Events
+- Pendo
+- Fivetran
+- AppsFlyer
+- Braze
+- Intercom
+
+### How to track
+
+Avo functions usage consists of 3 steps.
+
+#### Step 1. Include the Avo file 
+
+It can be done with the Avo CLI or manually
+
+##### Avo CLI is available through npm
+
+To install it and get the generated Avo file run the following commands.
+
+```bash
+$ npm install -g avo
+$ avo login
+$ avo pull --branch my-branch-name
+```
+
+Learn more about <Link passHref href="/implementation/cli"><a>the CLI here</a></Link>.
+
+##### To download the file manually navigate to Implementat tab in the side menu, then pick your source and press "Click to generate your Avo file" panel
+
+<center style={{marginBottom: 16 + "px"}}>
+  <img
+    src={require('../../../images/generate-avo-file-manually.png')}
+    alt="Implementation - Click to generate your Avo file"
+  />
+</center>
+
+Your file will be generated after that and you'll be able to download or copy it from there.
+
+#### Step 2. Initialize Avo
+
+Import Avo from the generated file
+
+```javascript
+import Avo from './Avo';
+```
+
+Before tracking anything with Avo you need to initialize it be calling the `initAvo` method, that looks similar to this
+
+```javascript
+function initAvo(options, systemProperties, destinationOptions, customDestDestination)
+```
+
+The actual parameters depend on your tracking plan setup, see the parameters explanation in <Link passHref href="/implementation/devs-101#avo-generated-file-init"><a>the initialization overview</a></Link>
+
+and you call it like this
+
+```javascript
+Avo.initAvo(
+  { env: 'dev', noop: false, strict: false, avoLogger: avoLogger, inspector: avoInspector },
+  { ...systemProperties },
+  { appsFlyer: { appId: 'XXXXXX' } },
+  customDestination
+);
+```
+
+#### About the custom destinations
+
+The last parameter in `initAvo` is special. If you choose to send your tracking to a custom destination in the Avo UI, it will be generated.
```

Comment:
```suggestion
If you choose to send your tracking to a custom destination in the Avo UI, a parameter for it will be added to the `initAvo` method.
```